src/service/getPersonasConnection.ts

```diff
@@ -10,8 +10,13 @@ import Config from './Config';
 
 export default (config: Config) => {
   return async (opts: GetPersonasConnectionOptions): Promise<GetPersonasConnectionResult> => {
+    if (opts.limit === undefined) {
```

Comment:
I'm not sure I understand the need for this change. The previous code would have handled this already, was there a typing issue?

Reply:
Yes, there is a test for if limit is undefined. According to the type, it should never be undefined. New tsc didn't like the spread as limit should always be defined according to its type.

Reply:
I see, @cbishopvelti should we just correct the type?